### PR TITLE
{tools}[GCCcore/11.2.0,GCCcore/11.3.0,GCCcore/12.2.0] jq v1.6

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bwd-rockylinux-90 - Linux Rocky Linux 9.0 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.10
See https://gist.github.com/b17a0da5a1ab1284962f316daee87310 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.6
See https://gist.github.com/a016b2064bb244200f4032d6c47d3de1 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
jrc0870.jureca - Linux Rocky Linux 8.6, AArch64, ARM UNKNOWN (graviton2), 2 x NVIDIA NVIDIA A100-PCIE-40GB, 515.65.01, Python 3.6.8
See https://gist.github.com/33a711abbde21f47566c24d0cbc12a4b for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 7 out of 7 (3 easyconfigs in total)
node151.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/37085f007d970646b040fafed1262ff3 for a full test report.

File: easybuild/easyconfigs/j/jq/jq-1.6-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+preconfigopts = "autoreconf -fi && "
+
+configopts = '--with-oniguruma=builtin'
+
+sanity_check_paths = {
+    'files': ['bin/jq'],
+    'dirs': [],
+}
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/j/jq/jq-1.6-GCCcore-11.2.0.eb:30]:
```suggestion

sanity_check_commands = ['jq --help', 'jq --version']

```

File: easybuild/easyconfigs/j/jq/jq-1.6-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+preconfigopts = "autoreconf -fi && "
+
+configopts = '--with-oniguruma=builtin'
+
+sanity_check_paths = {
+    'files': ['bin/jq'],
+    'dirs': [],
+}
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/j/jq/jq-1.6-GCCcore-11.3.0.eb:30]:
```suggestion

sanity_check_commands = ['jq --help', 'jq --version']

```

File: easybuild/easyconfigs/j/jq/jq-1.6-GCCcore-12.2.0.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+preconfigopts = "autoreconf -fi && "
+
+configopts = '--with-oniguruma=builtin'
+
+sanity_check_paths = {
+    'files': ['bin/jq'],
+    'dirs': [],
+}
+
```
Comment (jfgrimm) [inline easybuild/easyconfigs/j/jq/jq-1.6-GCCcore-12.2.0.eb:30]:
```suggestion

sanity_check_commands = ['jq --help', 'jq --version']

```

**Review by jfgrimm — COMMENTED** [review]
would be nice to add a sanity check command as well

**Comment by SebastianAchilles** [comment]
> would be nice to add a sanity check command as well

Good idea, thanks for the review :+1:

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bwd-rockylinux-90 - Linux Rocky Linux 9.0 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.10
See https://gist.github.com/f926e8642d94ce7fe9bfe1be8adeb2b7 for a full test report.

**Review by jfgrimm — APPROVED** [review]
LGTM

**Pull request merged**